Expose metrics to prometheus (#292)
cybozu-go/cke: docs/metrics.md

Metrics
=======

CKE exposes the following metrics with the Prometheus format. The listen address can be configured by the CLI flag (see [here](cke.md#Usage)). All these metrics are prefixed with `cke_`

| Name              | Description                                      | Type  | Labels                             |
| ----------------- | ------------------------------------------------ | ----- | ---------------------------------- |
| operation_running | True(=1) if any operations are running.          | Gauge |                                    |

> | boot_leader       | True(=1) if the boot server is the leader of CKE | Gauge |                                    |

Reviewer: `boot_` looks extraneous.

> | node_info         | The Control Plane and Worker info                | Gauge | address, rack, role, control_plane |

Reviewer: This is completely not a metric.

Metrics
=======

> CKE exposes the following metrics with the Prometheus format. The listen address can be configured by the CLI flag (see [here](cke.md#Usage)). The path is fixed to `/metrics`. All these metrics are prefixed with `cke_`

Reviewer: ```suggestion
CKE exposes the following metrics with the Prometheus format at `/metrics` REST API endpoint.  All these metrics are prefixed with `cke_`
```
